_layouts/default.html

```diff
@@ -2,66 +2,127 @@
 <html lang="en">
   <head>
     {% include head.html %}
-    <link rel="stylesheet" href="css/search.css">
-    <link rel="stylesheet" href="css/inner-page.css">
-    <meta name="description" content="LivePerson for developers: Build on the LivePerson Conversational Cloud with our powerful digital toolbox">
-    <meta name="keywords" content="APIs, SDKs, Mobile development, LiveEngage, Agent workspace, Tag, JSON examples, Click to message, Click to chat, Chat window, Skills, Agent Groups, iOS, Android, Mobile devices, Integrations, Live chat, Agent widget, Reporting, Analytics">
-    <meta name="author" content="LivePerson Inc.">
-    <link rel="stylesheet" href="https://cdn.jsdelivr.net/docsearch.js/2/docsearch.min.css">
+    <link rel="stylesheet" href="css/search.css" />
```

Comment:
Sorry I missed this (was not expanded in the view):

Entire file: Check and correct formatting—for some reason, there are many inner-tag breaks.

(If this was intended and you believe we should change this everywhere, let me know—but it seems this was some tooling accident?)